5 мелких фиксов

goodArthas opened the pull request.

Commit:
Изменена иконка приложения

Commit:
Исправлена текстовая ошибка в string файле

Commit:
-Вынесены string ресурсы с перезентеров TraderMeTradePresenter TraderTradePresenter
-Внедрен ResourceProvider в эти классы чтобы вытащить те же ресурсы string

Commit:
Изменен цвет положительного процента и графика

Commit:
Изменен внешний вид textInputLayout (style включен) и хинт к нему (изменен в String.xml)

Commit:
Merge branch 'develop' of https://github.com/WinTrade/WinTradeAndroid into bugfix/cheapFixes

FabulusTrade commented on app/src/main/java/ru/wintrade/util/Constants.kt:
```diff
@@ -5,4 +5,5 @@ const val HAS_VISITED_TUTORIAL = "hasVisitedTutorial"
-const val COLOR_GREEN = (0xFF00923B).toInt()
\ No newline at end of file
+const val COLOR_GREEN = (0xFF00923B).toInt()
+const val COLOR_GREEN_PERCENT = (0xff00923b).toInt()
```
зачем?! надо посмотреть используется ли COLOR_GREEN в проекте и убрать его в целом..ресурсы подтягивать через ResourceProvider а сами цвета разместить в colors

FabulusTrade commented on app/src/main/java/ru/wintrade/mvp/presenter/traderme/TraderMeProfitPresenter.kt:
```diff
@@ -212,7 +213,7 @@ class TraderMeProfitPresenter(
         val barDataSet = BarDataSet(entries, TraderProfitPresenter.PROFITABILITY)
         barDataSet.colors =
-            listOf(Color.GREEN, Color.BLACK, Color.RED)
+            listOf(COLOR_GREEN_PERCENT, Color.BLACK, Color.RED)
```
все цвета через ResourceProvider

FabulusTrade commented on app/src/main/res/layout/fragment_reset_password.xml:
```diff
@@ -15,12 +15,14 @@
         android:layout_marginTop="32dp"
         android:layout_marginEnd="20dp"
         android:hint="@string/email"
+        android:textColorHint="#66000000"
```
в ресурсы

FabulusTrade commented on app/src/main/java/ru/wintrade/ui/adapter/ObservationRVAdapter.kt:
```diff
@@ -37,7 +37,11 @@ class ObservationRVAdapter(val presenter: IObservationListPresenter) :
         if (holder.itemView.tv_subscriber_observation_profit.text.substring(0, 1) == "-") {
             holder.itemView.tv_subscriber_observation_profit.setTextColor(Color.RED)
```
Тоже из ресурсов тянуть

FabulusTrade commented on app/src/main/java/ru/wintrade/ui/adapter/CompanyTradingOperationsRVAdapter.kt:
```diff
@@ -35,7 +35,11 @@ class CompanyTradingOperationsRVAdapter(val presenter: ICompanyTradingOperations
                 holder.itemView.tv_item_comp_trading_ops_profit.setTextColor(Color.RED)
```
из ресурсов

FabulusTrade commented on app/src/main/java/ru/wintrade/mvp/presenter/trader/TraderProfitPresenter.kt:
```diff
@@ -169,7 +169,7 @@ class TraderProfitPresenter(
         val barDataSet = BarDataSet(entries, PROFITABILITY)
         barDataSet.colors =
-            listOf(COLOR_GREEN, Color.BLACK, Color.RED)
+            listOf(COLOR_GREEN_PERCENT, Color.BLACK, Color.RED)
```
Все через ResourceProvider

FabulusTrade commented on app/src/main/java/ru/wintrade/ui/adapter/TradersAllRVAdapter.kt:
```diff
@@ -30,7 +30,11 @@ class TradersAllRVAdapter(val presenter: ITradersAllListPresenter) :
         if (holder.itemView.tv_traders_all_item_year_profit.text.substring(0, 1) == "-")
             holder.itemView.tv_traders_all_item_year_profit.setTextColor(Color.RED)
```
из ресурсов

goodArthas committed:
В классы были добавлены ResourceProvider.kt через Inject где нужно было внедрить цвета через провайдер.
В аппкомпонент дагера были добавлены методы для inject вышеупомянутых классов
В ResourceProvider был добавелен метод для предоставления цветов
В colors.xml добавлены цвета для:
-Хинта textview
-цветовой схемы BarChart
-цветов процентов

goodArthas committed:
Merge branch 'develop' of https://github.com/WinTrade/WinTradeAndroid into bugfix/cheapFixes

 Conflicts:
	app/src/main/java/ru/wintrade/mvp/presenter/trader/TraderProfitPresenter.kt
	app/src/main/java/ru/wintrade/mvp/presenter/traderme/TraderMeProfitPresenter.kt

goodArthas committed:
Конфликты устранены
